### PR TITLE
Exclude code paths that do not required v2 check

### PR DESCRIPTION
**Pull request by rukawata**
Explicitly exclude paths that do not need to be checked by azure-pipeline-v2.

### CODE REVIEW

File: azure-pipelines-v2.yaml [hunk 1]
```diff
@@ -10,6 +10,10 @@ pr:
   paths:
     include:
     - deploy/v2/*
+    exclude:
```
Comment (tniek) [inline azure-pipelines-v2.yaml]: Not quite sure on the logic here:
So we're explicitly including v2 only - but at the same time excluding others that are already mutually exclusive anyway? Why is this needed?

Comment (rukawata) [reply azure-pipelines-v2.yaml]: > Not quite sure on the logic here:
> So we're explicitly including v2 only - but at the same time excluding others that are already mutually exclusive anyway? Why is this needed?

I agree this is confusing. But simply include deploy/v2/* does not exclude other folders...that is why this PR is needed.
